tests/suites/0_stateless/09_0007_func_fuse_truncate_history.sql

```diff
@@ -8,41 +8,15 @@ insert into t values (5);
 insert into t values (6);
 insert into t values (7);
 
--- expects 3
+-- expects 3 history items
 select count(*) from fuse_history('db_09_0007', 't');
 
-
--------primary case--------------
-
--- do the truncation (without syntax sugar)
--- 2 snapshot removed, and not segments or blocks will be removed
-select * from fuse_truncate_history('db_09_0007', 't');
--- after truncation, count of history will 1
+-- truncate table will remove all the historical data but the latest one
```

Comment:
If the latest one snapshot has datas after `truncate table 't'`, does `select * from t1` has the results?

Reply:
`select *  from t1`  returns an empty resultset in this case, the semantic of the statement `truncate table` does not change. 

but a snapshot will be kept, which references no segments or blocks.